### DISCUSSION TITLE
How do I set the sender header?

### DISCUSSION THREAD

**Discussion by ghost** [postalserver/postal#1740] [Help with using Postal] [answered]
Hi! I'm trying to send emails with Postal from my domain, and I've enabled "Allow Sender Header" and I'm making a request with both the `sender` and `from` keys like it says to in the [API docs](http://apiv1.postalserver.io/controllers/send/message), but it still displays in my mail client as `x8x2qd@psrp.allio.cloud` rather than `test@allio.cloud` for the sender header.
![image](https://user-images.githubusercontent.com/42045366/143683805-fcbfc496-d2ac-4594-ab65-b6764271c372.png)
![image](https://user-images.githubusercontent.com/42045366/143683899-1b8ca0de-fa2f-46be-a3f9-6bd19e8bfded.png)

Is it possible to use Postal's API to just send an email with both as the user-readable email address? Like how emails from other services display with just the "From" address:
![image](https://user-images.githubusercontent.com/42045366/143684044-900d90ad-0a39-41b7-a09a-8dccfad1b74c.png)

**Comment by willpower232** [comment, marked as answer]
If your from and sender headers are the same then you do not need to use the sender header. It would be nice if those docs mentioned that but thats something else.

It looks like your email client is displaying the return path email address for some reason. This email address allows Postal to handle bounces if the email address does not exist and is not typically shown to the recipient.

Out of curiousity, what email client are you using?

**Reply by ghost** [reply]
> If your from and sender headers are the same then you do not need to use the sender header

I know, I was just using it in that example to show that it didn't have an affect on how my mail client displayed it.

I'm using [Tutanota](https://tutanota.com/)'s mail client, and yeah, it seems that it does have the correct sender if I look at the headers section in the app, but it displays the return path normally, which is odd.

**Reply by willpower232** [reply]
Judging by their stance on security, I'd imagine that they are displaying the reply-to email address to make it more obvious where any reply would go, that can be an easy way of spotting spam messages but I don't think other email clients display it in that way so you're probably fine unless you are only emailing people who use Tutanota.
